charts/cloudprem/values.yaml

```diff
@@ -59,9 +59,10 @@ securityContext:
 # Additional global env
 environment:
   QW_DISABLE_TELEMETRY: true
-  QW_ENABLE_OPENTELEMETRY_OTLP_EXPORTER: true
   NO_COLOR: true
-  OTEL_EXPORTER_OTLP_ENDPOINT: http://$(HOST_IP):4317
+  # Indexing Pomsky traces within Pomsky
+  QW_ENABLE_OPENTELEMETRY_OTLP_EXPORTER: true
+  OTEL_EXPORTER_OTLP_ENDPOINT: "http://localhost:7281"
```

Comment:
@fmassot had concerns about this https://github.com/DataDog/helm-charts/pull/1848#discussion_r2068839326

Reply:
we are good on this